metadata.py

```diff
@@ -40,27 +40,27 @@ def appmetadata() -> AppMetadata:
         # analyzer_license="",  # short name for a software license
     )
     # and then add I/O specifications: an app must have at least one input and one output
-    metadata.add_input(AnnotationTypes.TimeFrame)  # Output from SWT
-    metadata.add_input(DocumentTypes.TextDocument) # Output from OCR
+    metadata.add_input(AnnotationTypes.TimeFrame)   # Output from SWT
+    metadata.add_input(DocumentTypes.TextDocument)  # Output from OCR
     metadata.add_output(DocumentTypes.TextDocument)
     metadata.add_output(AnnotationTypes.Alignment)
     
-    
     # (optional) and finally add runtime parameter specifications
-    metadata.add_parameter(name='contain_labels',
+    metadata.add_parameter(name='containLabels',
                            description=
                            '''
-                           A list of labels of TimeFrame to look for.
+                           A list of labels that user expect TimeFrames contain.
                            Available options are:
                            ['bars', 'tones', 'bars-and-tones','speech',
                            'noise', 'music', 'slate', 'chyron',
                            'lower-third', 'credits']
-                           User needs to provide a string of comma-separated labels, 
-                           e.g. 'bars, credits, chyron'.
-                           Default is all are selected.
+                           User needs to provide a list of labels if multiple labels are expected
```

Comment:
This is not really true. For multivalued parameters, users need to do something like [this](https://github.com/clamsproject/clams-python/issues/122#issuecomment-1513424021) . (SDK will make sure the `_annotate` method receives a list from that URL string format)

Reply:
And, this URL format should be something users need to know first (or learn from the general app manual), so that doesn't need to be fully spelled out here, I guess. Let's just take out the format-related explanation.

Reply:
> For multivalued parameters, users need to do something like this

Does this mean that we expect users to run our app as `curl -X ... localhost:xxxx?containLabels=credits&containLabels=slate` if they're interested in `TimeFrame`s that contain `label` property "credits" and "slate"?

Reply:
Yes, exactly.